app/models/katello/event_queue.rb

```diff
@@ -0,0 +1,25 @@
+module Katello
+  class EventQueue < Katello::Model
```

Comment:
My object oriented brain feels like this should be an Event with a class that represents the EventQueue and manages the events vs an EventQueue managing instances of event queues.